src/navigation/tabs/tabs.jsx

```diff
@@ -1,86 +1,674 @@
+
+        if (reducedItems.isSelectedTabHidden) {
+            const lastVisibleTab = reducedItems.tabsVisible[reducedItems.tabsVisible.length - 1];
+            const hiddenSelectedTab = find(reducedItems.tabsHidden, { selected: true });
+            const hiddenSelectedTabIndex = findIndex(reducedItems.tabsHidden, { selected: true });
+
+            reducedItems.tabsVisible.splice(
+                reducedItems.tabsVisible.length - 1,
+                1,
+                hiddenSelectedTab,
+            );
+
+            reducedItems.tabsHidden.splice(hiddenSelectedTabIndex, 1, lastVisibleTab);
+            sortedHiddenTabs = sortBy(reducedItems.tabsHidden, ['key']);
+            sortedVisibleTabs = sortBy(reducedItems.tabsVisible, ['key']);
+
+            const reSortedItems = [
+                ...reducedItems.tabsVisible,
+                ...sortedHiddenTabs,
+            ]
+                .filter((tab) => !!tab) // makes sure there's no undefined tab when several resize events happen rapidly
+                .map((tab) => find(items, { key: tab.key }));
+
+            this.setState({
+                items: reSortedItems,
+            });
+        }
+
+        reducedItems = {
+            ...reducedItems,
+            isSelectedTabHidden: false,
+            tabsHidden: sortedHiddenTabs,
+            tabsVisible: sortedVisibleTabs,
+        };
+
+        return reducedItems;
+    }
+
+    getTabProps(tab = {}) {
+        const {
+            classes,
+            mobile,
+            inverse,
+            withContent,
+        } = this.props;
+
+        const {
+            className,
+            collapsed,
+            disabled,
+            key,
+            onClick,
+            selected,
+            tabIndex,
+            title,
+        } = tab;
+
+        return {
+            children: title,
+            classes: {
+                label: classes.label,
+                mobile: classes.mobile,
+                selected: classes.selected,
+                withContent: classes.withContent,
+            },
+            classNames: this.getClassNamesFor('tab', {
+                className,
+                collapsed,
+                disabled,
+                selected,
+                tabIndex,
+            }),
+            id: PREFIX_TAB + key,
+            inverse,
+            key: PREFIX_TAB + key,
+            mobile,
+            originalKey: key,
+            onClick,
+            onChange: this.onChangeTab,
+            ref: (ref) => { this.tabRefs[PREFIX_TAB + key] = ref; },
+            selected,
+            withContent,
+        };
+    }
+
+    getPanelProps(panel, isHidden) {
+        const {
+            className,
+            content,
+            getContent,
+            key,
+        } = panel;
+
+        return {
+            getContent,
+            children: content,
+            key: CONTENT_PREFIX + key,
+            id: CONTENT_PREFIX + key,
+            tabId: PREFIX_TAB + key,
```

Comment:
We should use interpolation rather than concatenation here, right?

Reply:
Eslint is not bugged by this, but I can use a template literal.

Reply:
Changed

Reply:
Probably 'cause ESLINT isn't quite smart enough to realize that the left-hand side is a string when it's a symbolic constant like that (versus a string literal) :-P

In any event, fixed.
I noticed a few other places up on Lines 456 and 458 as well, but those are _TOTALLY NABS._
Resolving.